### DISCUSSION TITLE
Clip Skip and Vae

### DISCUSSION THREAD

**Discussion by MxtrSplinter** [Acly/krita-ai-diffusion#480] [Q&A]
I'm trying to use pony diffusion, which requires the use of clip skip and a VAE. Is there a way of accessing those via the plugin interface? Should i open the Comfy Workflow related to the style and add the nodes myself?

**Comment by Acly** [comment]
Clip skip and vae can be configured in Style settings (advanced checkpoint options).
I think default clip skip (2 for SDXL) should be fine though.

**Reply by MxtrSplinter** [reply]
Thanks for your quick answer! I had'nt noticed the options to change them in the collapsible menu. In the end, turns out Pony diffusion had other issues that were causing the bad generations, in this case, incompatibilities with Control.Net, that I hope they will fix soon!
Also, Thanks so much for this plugin! It works great!

**Comment by tommywright** [comment]
It's not Clip Skip that we need for Pony.. we need to disable the clip_g and clip_l and load that stuff through the Checkpoint. When I make a Pony Workflow, I use a Load Checkpoint node that has Model and Clip in that node. I'm not sure if it's built in or what but I use the Clip pin straight out of that Load Checkpoint. If I use clip_g and clip_l, I get terrible results.
